templates/claude/agents/sm.md

```
@@ -139,54 +142,59 @@ if [ -z "${BUFFER_DIR:-}" ]; then
+   - 新しい順で上に追加する。
+   - `decisions-index.md` が存在する場合は追記する。
+   - `decisions-index.md` と `decisions.md` が両方不在の場合のみ新規作成する（テンプレートと同形式）。
 
 **書き込み順序**: アーカイブ → インデックスの順で書く。アーカイブ成功後に index 追記が失敗しても、次回 step 1 で index エントリ欠落を検知し補完できる（逆順だと index のみ更新され archive が無い不整合になる）。
 
 `BUFFER_DIR` が空（フォールバック失敗）の場合: 判断内容を本レビュー結果の「`### 判断記録（記録先取得失敗）`」セクションに含めて呼出元に返却し、人間または呼出元側で手動反映する。これにより無言データロストを回避する。
 
-**ヘッダー名は厳格指定**: 必ず正確な文字列「`### 判断記録（記録先取得失敗）`」で出力する。バリエーション（例: `## フォールバック判断記録`、`### 判断記録 (記録失敗)`、半角カッコ等）は禁止する。呼出元スキルはこのヘッダーで grep して検知するため、表記ゆれは検知漏れにつながる。
+> [!WARNING]
+> **ヘッダー名は厳格指定**: 必ず正確な文字列「`### 判断記録（記録先取得失敗）`」で出力する。バリエーション（例: `## フォールバック判断記録`、`### 判断記録 (記録失敗)`、半角カッコ等）は禁止する。呼出元スキルはこのヘッダーで grep して検知するため、表記ゆれは検知漏れにつながる。
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)